pyanaconda/modules/payload/shared/initialization.py

```diff
@@ -0,0 +1,79 @@
+#
+# Copyright (C) 2019 Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+import os
+import shutil
+from glob import glob
+
+from pyanaconda.core.constants import DD_ALL, DD_FIRMWARE, DD_RPMS
+from pyanaconda.core.util import getSysroot
+from pyanaconda.modules.common.task import Task
+from pyanaconda.modules.payload.shared.utils import create_root_dir, write_module_blacklist
+
+
+from pyanaconda.anaconda_loggers import get_module_logger
+log = get_module_logger(__name__)
+
+
+class PrepareSystemForInstallation(Task):
```

Comment:
Please, add `Task` at the end of the name. It is a common practice.

Reply:
Fixed.